tests/test_transpile.py

```diff
@@ -46,7 +46,7 @@ def test_alias(self):
             "SELECT 1 FROM a.b.table1 AS t UNPIVOT((c3) FOR c4 IN (a, b))",
         )
 
-        for key in ("union", "over", "from", "join"):
+        for key in ("union", "from", "join"):
```

Comment:
I didn't observe any weird behavior, seems like we can still parse this just fine. The original [commit](https://github.com/tobymao/sqlglot/commit/cbe099c39650fd31797739d88c19c46597263b4a) that introduced this didn't include any tests and the issue didn't include `OVER` in the repro.